benchmarking/collect_netlib_benchmark.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# Copyright 2021 The FirstOrderLp Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script illustrates how to collect the instances in the "Netlib Benchmark"
+# set. Although it can be used directly to collect the benchmark, it is intended
+# primarily as a guide to collecting the benchmark - it makes several
+# assumptions about the environment. In particular, it assumes wget, gzip, f77,
```

Comment:
Perhaps use gfortran instead of f77 to make it clearer which package should be installed.

Reply:
Done.